styles.css

```diff
@@ -0,0 +1,600 @@
+  width: 40%;
+  padding-left: 30%;
+  padding-right: 30%;
+}
+
+#title {
+  display: inline-block;
+  font-size: 48px;
+  padding-top: 50px;
+}
+
+/* +++ Article +++ */
+
+.titlestyle {
+  font-weight: bold;
+  font-size: 30px;
+}
+
+.artpics1 {
+  display: grid;
+  grid-template-columns: 15% 70% 15%;
+}
+
+#one {
+  grid-column-start: 2;
+  max-width: 100%;
+  height: auto;
+}
+
+#onefig {
+  grid-column-start: 2;
+  color: #666;
+}
+
+#oneauth {
+  grid-column-start: 2;
+  display: inline-flex;
+}
+
+#dennis {
+  color: #333;
+}
+
+#denniso {
+  color: #333;
+}
+
+#denniso:hover {
+  text-decoration: none;
+}
+
+#datsoc {
+  display: flex;
+  justify-content: space-between;
+}
+
+#clear {
+  clear: both;
+}
+
+.artcont {
+  display: grid;
+  grid-template-columns: 27.5% 45% 27.5%;
+  text-align: left;
+  clear: both;
+}
+
+.article {
+  grid-column-start: 2;
+}
+
+.article > p > a {
+  color: #326891;
+  text-decoration: underline;
+}
+
+.article > p > a:hover {
+  text-decoration: none;
+}
+
+.artpics2 {
+  display: grid;
+  grid-template-columns: 15% 70% 15%;
+}
+
+#inflatt {
+  grid-column-start: 2;
+  font-size: 25px;
+}
+
+#inflagrid {
+  grid-column-start: 2;
+  display: grid;
+  grid-template-areas:
+    "im11 im12 im13 im14"
+    "par11 par12 par13 par14"
+    "im21 im22 im23 im24"
+    "par21 par22 par23 par24";
+}
+
+#inflation {
+  grid-area: 1 / 1 / 5 / 5;
+  max-width: 100%;
+}
+
+#inflationfig {
+  grid-column-start: 2;
+}
+
+.bold {
+  font-weight: bold;
+}
+
+#par11 {
+  grid-area: par11;
+  font-size: 15px;
+}
+
+#par12 {
+  grid-area: par12;
+  font-size: 15px;
+}
+
+#par13 {
+  grid-area: par13;
+  font-size: 15px;
+}
+
+#par14 {
+  grid-area: par14;
+  font-size: 15px;
+}
+
+#par21 {
+  grid-area: par21;
+  font-size: 15px;
+}
+
+#par22 {
+  grid-area: par22;
+  font-size: 15px;
+}
+
+#par23 {
+  grid-area: par23;
+  font-size: 15px;
+}
+
+#par24 {
+  grid-area: par24;
+  font-size: 15px;
+}
+
+#inflafoot {
+  grid-column-start: 2;
+  font-size: 18px;
+}
+
+#floatleft {
+  float: left;
+  width: 27.5%;
+  background-color: #fff;
+}
+
+#hidden-paragraph {
+  visibility: hidden;
+}
+
+#thwrapper {
+  float: left;
+  width: 45%;
+}
+
+#thcont {
+  text-align: center;
+}
+
+#thanks {
+  text-align: center;
+  clear: both;
+}
+
+#thanks > a {
+  color: #326891;
+  text-decoration: none;
+  font-size: 24px;
+}
+
+.floatingarticle {
+  float: left;
+  width: 45%;
+}
+
+#asideboxmt {
+  float: right;
+  width: 20%;
+  padding-right: 30px;
+}
+
+#asideitems {
+  text-align: left;
+}
+
+.asideimg {
+  width: 75px;
+  float: left;
+}
+
+.asidelink {
+  color: #797979;
+  text-decoration: none;
+  padding: 4px 4px 4px 4px;
+}
+
+.asidelink:hover {
+  color: #888;
+  text-decoration: underline;
+}
+
+.nytline {
+  size: 10%;
+}
+
+#telescope-cont {
+  text-align: center;
+}
+
+#telescope {
+  width: 70%;
+}
+
+.videochamp {
+  min-width: 100%;
+  width: auto;
+  border: none;
+}
+
+#lastartlinkscont {
+  text-align: center;
+}
```

Comment:
Same style definition of the selector on line 334. Group selectors to minimize your code.